src/registry_schemas/schemas/business.json

```diff
@@ -133,6 +152,29 @@
           "default": "",
           "examples": ["123456789"],
           "pattern": "^[0-9]{9}$"
+        },
+        "state": {
+          "type": "string",
+          "title": "The state of the business",
+          "enum": [
+            "ACTIVE",
+            "HISTORICAL",
+            "LIQUIDATION"
+         ]
+        },
+        "stateFiling": {
+          "type": "string",
+          "title": "The link to the filing that changed the business state (if applicable)s."
+        },
+        "goodStanding": {
+          "title": "Indicates whether the company is in good standing.",
+          "type": "boolean"
+        },
+        "complianceWarnings": {
```

Comment:
Will this always be provided, but perhaps be an empty array?

Reply:
Can be done that way

Reply:
I'd like to know, for parsing in the UI, whether this could be undefined/null, or `[]`.

Reply:
My thought is to include it in the response only if there are warnings. So it could be null/undefined

Reply:
OK